### PR TITLE
added configuration file analysis blog

### PR DESCRIPTION
**Pull request by SankBad**
This PR will add entry in toc.yaml and content-sources.yaml files for configuration file analysis blog. 
We used remote repo([src](https://github.com/aicoe-aiops/configuration-files-analysis)) for adding this blog. 
I tested the site locally and it looks as expected.

### CODE REVIEW

**Review by oindrillac — CHANGES_REQUESTED** [review]
@SankBad I get an error from the ssh url in the content-sources.yaml (when I try to build without manually cloning the repo). Can you replace it with https://github.com/aicoe-aiops/configuration-files-analysis.git?

**Comment by SankBad** [comment]
@oindrillac , Thanks for pointing out.
I changed ssh key to a web url.

**Comment by oindrillac** [comment]
@SankBad I think the ignore key added in #48 for the continuous-deployment repo got moved. can you please move it back?

**Comment by SankBad** [comment]
Hi @oindrillac , 

ignore key is available in this PR. 
I rebase my repo before making the changes.

File: content-sources.yaml [hunk 1]
```diff
@@ -14,5 +14,10 @@
   gitSrc: https://github.com/operate-first/continuous-deployment.git
   dir: continuous-deployment/docs
   urlPrefix: cd
+
+- name: configuration file analysis
+  gitSrc: https://github.com/aicoe-aiops/configuration-files-analysis.git
+  dir: configuration-files-analysis/docs/blog
+  urlPrefix: config_analysis
```
Comment (cfchase) [inline content-sources.yaml]: I would stick with dash `config-analysis` instead of `config_analysis` in urls just to be consistent

**Review by cfchase — CHANGES_REQUESTED** [review]

**Comment by SankBad** [comment]
@cfchase @oindrillac 
Thanks for the feedback. 
I fixed the format of urlPrefix and changed the location of ignore key.
Also, squashed the commits.

File: content-sources.yaml [hunk 2]
```diff
@@ -16,3 +16,9 @@
   urlPrefix: cd
   ignore:
     - '**/README.md'
+
+- name: configuration file analysis
+  gitSrc: https://github.com/aicoe-aiops/configuration-files-analysis.git
+  dir: configuration-files-analysis/docs/blog
+  urlPrefix: config-analysis
+
```
Comment (cfchase) [inline content-sources.yaml:24]: add `configuration-files-analysis`  in the Remote Content section in the [`.gitignore`](https://github.com/operate-first/operate-first.github.io/blob/master/.gitignore#L2)

**Review by cfchase — CHANGES_REQUESTED** [review]

**Comment by SankBad** [comment]
Hi @cfchase ,

Added entry in the [`.gitignore`](https://github.com/operate-first/operate-first.github.io/blob/master/.gitignore#L2) file

**Review by cfchase — APPROVED** [review]

**Review by oindrillac — APPROVED** [review]

**Pull request merged**